test/merchant_repository_test.rb

```diff
@@ -78,6 +78,16 @@ def test_most_revenue_returns_top_x_merchant_instances_ranked_by_total_revenue
     assert_equal  3, result_2
     assert_equal 30, result_3
   end
+  
+  def test_most_items_returns_top_x_merchant_instances_ranked_by_total_items_sold
```

Comment:
Assignment Branch Condition size for test_most_items_returns_top_x_merchant_instances_ranked_by_total_items_sold is too high. [15.3/15]<br>Line is too long. [81/80]